modules/calendar/add_edit_event.php

```diff
@@ -230,6 +230,28 @@ function DOBandEncounter()
         if ($_POST['form_ampm'] == '2' && $tmph < 12) $tmph += 12;
         $duration = abs($_POST['form_duration']); // fixes #395
     }
+    //check for selected appointment time
+    if ($GLOBALS['check_appt_time'] == '1') {
```

Comment:
I had to make this if ($GLOBALS['check_appt_time'] == 1) to get it to work on my system

Reply:
@teryhill I thought this global was carrying the string 1 rather than numeric 1 when checked. Apparently, that's not the case.